src/main/scala/no/ndla/draftapi/service/WriteService.scala

```diff
@@ -432,6 +438,9 @@ trait WriteService {
           isImported = externalIds.nonEmpty,
           shouldAlwaysCopy = updatedApiArticle.createNewVersion.getOrElse(false)
         )
+        _ <- partialPublish(existing.id.get,
```

Comment:
Selvom jeg vet vi er late på `.get` andre steder så liker jeg å prøve å holde det så lite som mulig.
Sleng inn en match sånn som i `updateTaxonomyForArticle` i samme fil feks?

Reply:
Liker ikke `.get` selv og er 100% enig, men akkurat i dette tilfellet så er det kanskje greit? Denne funksjonen blir kallt slik: 

```scala
    def updateArticle(articleId: Long,
                      ... ): Try[api.Article] =
      draftRepository.withId(articleId) match {
        case Some(existing) =>
          updateExistingArticle(
            existing,
             ...
          )
```

Så `existing` er en artikkel direkte fra databasen og etter min forstårelse så har alle artikkelene en `id` i databasen. Kan skrive om `.get` hvis vi vil innføre en god praksis og slutte å bruke `.get`, men dersom noe er så si garrantert å finnes så burde det være greit å bruke `.get` i de tilfellene kanskje?

Reply:
Meeeen hvis `None.get` vil forårsake at hele applikasjonen tryner på samme måte som å aksessere en nullpointer i c++, så er jeg helt med for å aldri bruke `.get` (uten om tester)